react/PWAContext.js

```diff
@@ -54,10 +54,14 @@ const PWAProvider = ({ rootPath, children, data = {} }) => {
   const context = useMemo(() => {
     if (pwaSettings) {
       const { disablePrompt, promptOnCustomEvent } = pwaSettings
+      const isIOS = navigator && !!navigator.userAgent.match(/(iPod|iPhone|iPad)/)
+      const installDismissed = JSON.parse(localStorage.getItem('appInstallDismissed'))
+      const alreadyInstalled = window.matchMedia('(display-mode: standalone)').matches
```

Comment:
do we need to add some guard before accessing the `window`? or do we know this code will always run on the client?

Reply:
No, we don't know. I'll add a guard for this.

Also, I forgot that the display mode can be changed :( maybe I should add a query param in the `start_url` on the manifest.json and check this way?

Reply:
yeah, I think it's the best way to do. and it could help with analytics too, because we would know which users are using the app instead of the website.